aq_zombie_finder/aq_zombie_finder.py

```diff
@@ -0,0 +1,117 @@
+import paramiko
+from re import compile
+from getpass import getpass
+
+
+# Creates an SSH client through Paramiko with the users details and returns the client
+def create_client(host, user, password):
```

Comment:
you've got some good comments here, but I would recommend writing docstrings instead of comments 

```suggestion
def create_client(host, user, password):
"""
Function to create an SSH client through Paramiko
    :param host: hostname to connect to (string)
    :param user: user to connect as (string)
    :param password: password for the user (string)
    :returns:  Configured Paramiko SSHClient object 
 """
```
A docstring is a comment that describes what the function does, what inputs it takes, and what it returns. I'd recommend changing your comments into docstrings